src/rollbar.php

```diff
@@ -237,6 +268,35 @@ public function queueSize() {
     }
 
     /**
+     * Run the checkIgnore function and determine whether to send the Exception to the API or not.
+     *
+     * @param  bool             $isUncaught
+     * @param  RollbarException $caller_args [level, message, err, custom]
+     * @param  array            $payload     Data being sent to the API
+     * @return bool
+     */
+    protected function _shouldIgnore($isUncaught, RollbarException $caller_args, array $payload)
+    {
+        try {
+            if (is_callable($this->checkIgnore)
+                && call_user_func_array($this->checkIgnore, array($isUncaught,$caller_args,$payload))
```

Comment:
Hm. So `RollbarException` seems to be duplicating a lot of stuff that's already present in `$payload`. Is there a notable difference?